agent/keysplitting/util.go

```diff
@@ -354,11 +360,13 @@ func (k *KeysplittingHelper) verifyIdToken(rawtoken string, cert kysplContracts.
 	case "None":
 		// If there is no provider, skip id token verification
 		// Provider isn't stored for single-player orgs
-		return time.Time{}, nil
+		return time.Now().Add(bzecertLifetime), nil
 	case "google":
 		issUrl = k.googleIss
 	case "microsoft":
 		issUrl = k.microsoftIss
+	case "okta":
+		issUrl = k.oktaIss
 	default:
 		issUrl = k.provider
```

Comment:
Maybe just throw an error here now instead of using k.provider as default issUrl